.github/workflows/test.yml

```diff
@@ -64,9 +64,9 @@ jobs:
           toxenv: py
           tox_extra_args: "-n 2"
         - name: mypyc runtime tests with py36-macos
-          python: '3.6'
+          python: '3.10'
```

Comment:
Why does the Python need to be changed? Regardless, the name should be consistent with the Python version